objects/oArrow/Collision_a4ad2bf4-ea42-4e71-845b-51c25d8831e8.gml

```diff
@@ -0,0 +1,2 @@
+other.currentHealth -= 1;
+instance_destroy();
```

Comment:
Maybe the fireball could be destroyed a frame or two after it hits the enemy? Just because atm it kind of gets destroyed just before it hits the enemy most of the time. Could use the same kind of firstFrame logic that you've got above